scripts/jenkins/Jenkinsfile.release

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env groovy
+
+// Imports
+import java.text.SimpleDateFormat
+
+// Buildjob properties
+properties([
+  buildDiscarder(
+    logRotator(
+      artifactDaysToKeepStr: '31',  // Keep artifacts for max 31 days
+      artifactNumToKeepStr: '10'    // Keep artifacts for last 5 builds
```

Comment:
comment inconsistent to actual setting. The name of the variable is also quite clear, so comment is not really needed.